packages/async-storage/webui/src/AsyncStorageTable.tsx

```diff
@@ -95,8 +95,8 @@ export function AsyncStorageTable() {
                 },
               };
             },
-            shouldCellUpdate(record, prevRecord) {
-              return record.editedValue !== prevRecord.editedValue;
+            shouldCellUpdate() {
+              return false;
             },
           },
           {
```

Comment:
```suggestion
                onBlur() {
                  setEditingManually(false);
                },
                onFocus() {
                  setEditingManually(true);
                },
              };
            },
            shouldCellUpdate(record, prevRecord) {
              return !editingManually && record.editedValue !== prevRecord.editedValue
            },
          },
```

I added a state to check whether the user currently has focused the cell and that does seem to fix the jump issue while still keeping the json editor working

Reply:
My bad, I didn't realize my change would break the json editor. I'll merge in this suggestion!

Reply:
I've updated the pr, like you said the json editor seems to be working now.

Reply:
> I've updated the pr, like you said the json editor seems to be working now.

In that case this looks good to me, so long as @Kudo doesn't have any issues